### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by Jakllp**
I noticed some bugs so I set out to fix them. This is what I did:

Letting roller coasters roll again
Storagecarts etc drop correctly now
Better configs (e.g. black_wool instead of WOOL:15 as roadblocks)

### CODE REVIEW

File: src/main/java/com/useful/ucars/ucars.java [hunk 1]
```diff
@@ -688,16 +688,25 @@ public final Boolean isBlockEqualToConfigIds(List<String> rawIds, Block block) {
...
-				if (mat.equalsIgnoreCase(block.getType().name()) && bdata == data) {
+				//TODO Alternative to .getData()
+				if (ItemStackFromId.equals(raw,block.getType().name().toUpperCase(),block.getData())) {
 					return true;
 				}
 			}
 		}
 		return false;
 	}
+	public final Boolean isBlockEqualToConfigIds(List<String> rawIds, String blockName) {
```
Comment (storm345) [inline src/main/java/com/useful/ucars/ucars.java:699]: Ought to support or convert existing config values. Perhaps prefer to convert to standard bukkit itemstack config representation

File: src/main/resources/plugin.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 name: uCars
 main: com.useful.ucars.ucars
-version: 20
+version: 20.75
+api-version: 1.16
```
Comment (storm345) [inline src/main/resources/plugin.yml:4]: The master branch of uCars should be 1.12 compatible as the primary version for now. 1.16 can be a separate branch, or can also be supported in addition to 1.12 but 1.12 must be compatible.

**Review by storm345 — CHANGES_REQUESTED** [review]
Had a look through and made some comments that will need addressing before it can be merged.

Looks like a bunch of this is updating for 1.16 whereas the master branch of uCars is for 1.12 currently. Ideally if the change can support both would be ideal, or if not then it ought to be a different branch for 1.16. 

There also seem to be a few small potential bugs - such as assuming the car is a Minecart and ignoring a custom hitbox for collisions.

It also would be a lot better to support or convert existing configurations and their definitions of material:data for types.

File: src/main/java/com/useful/ucars/uCarsListener.java [hunk 3]
```diff
@@ -260,23 +260,45 @@ public boolean inACar(String playername) {
...
 		Block b = loc.getBlock();
 		String mat = b.getType().name().toUpperCase();
+		String behindmat = getBehindBlock(cart);		
 		String underMat = b.getRelative(BlockFace.DOWN).getType().name().toUpperCase();
 		String underUnderMat = b.getRelative(BlockFace.DOWN, 2).getType().name().toUpperCase();
 		List<String> checks = new ArrayList<String>();
 		if(ucars.ignoreRails){
 			checks.add("POWERED_RAIL");
-			checks.add("RAILS");
+			checks.add("RAIL");
```
[the remark this answers is not in the document]
Comment (Jakllp) [reply src/main/java/com/useful/ucars/uCarsListener.java:297]: I assume you mean MC-VERSIONS?

**Pull request closed without merging**